internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/TestPushJobWithNativeReplication.java

```diff
@@ -458,7 +458,8 @@ public void testNativeReplicationForHeartbeatSystemStores(int recordCount, int p
             TestWriteUtils.updateStore(
                 VPJ_HEARTBEAT_STORE_NAME,
                 parentControllerClient,
-                new UpdateStoreQueryParams().setLeaderFollowerModel(true).setNativeReplicationEnabled(true));
+                new UpdateStoreQueryParams().setLeaderFollowerModel(true).setActiveActiveReplicationEnabled(true));
+            parentControllerClient.emptyPush(VPJ_HEARTBEAT_STORE_NAME, "new push", 1L);
```

Comment:
Not sure if you expect to wait for the push to complete. `sendEmptyPushAndWait` is a function if you want to wait for the push to HB store to complete

Feel free to drop.